scripts/components/RecentChanges/RecentChanges.js

```diff
@@ -5,11 +5,7 @@ import { useScrollNearEnd } from '../../hooks'
 import SkeletonChangeSet from './SkeletonChangeSet'
 import ChangeSet from './ChangeSet'
 
-const HISTORY_ENDPOINT =
-  window.location.host.endsWith('now.sh') ||
-  process.env.NODE_ENV === 'development'
-    ? 'https://api.mousey.app/v3/emoji/test/changes'
-    : 'https://api.mousey.app/v3/emoji/blobs+community-blobs/changes'
+const HISTORY_ENDPOINT = 'https://api.mousey.app/v3/emoji/blobs+community-blobs/changes'
```

Comment:
Why are we changing this? I was under the impression that this was needed for when either @mralext20 deploys it to `now.sh` or we are dev testing it.

Reply:
The master branch has moved to only a single endpoint, the now.sh problem was apparently solved